data/data/agent/systemd/units/node-zero-monitor.path

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Monitor the presence of the node0 file
+
+[Path]
+PathExists=/etc/assisted-service/node0
+Unit=assisted-service-pod.service
```

Comment:
Shouldn't this unit then not be enabled and only have the .path enabled?

Reply:
Thanks for the suggestion, initially I thought the `[Install]` section of `assisted-service-pod.service` was still required to let the dependencies work properly, but testing it (with a simulation of delayed node0 generation by a couple minutes) worked properly